Metadata/MetadataConverter.php

```diff
@@ -64,6 +64,14 @@ public function convert(ClassHierarchyMetadata $metadata)
             $definition->setTags($classMetadata->tags);
             $definition->setProperties($classMetadata->properties);
 
+            if (null !== $classMetadata->decorates && method_exists($definition, 'setDecoratedService')) {
+                $definition->setDecoratedService($classMetadata->decorates, $classMetadata->decoration_inner_name);
```

Comment:
Why would you want a weird behavior in your project ? 
That would mean that when using 2.8 your service will extend another one and in 2.3 it won't...